packages/ruby/spec/readme/har/response_serializer_spec.rb

```diff
@@ -105,6 +105,63 @@
       expect(json.dig(:content, :size)).to eq 0
       expect(json.dig(:content, :mimeType)).to eq ""
     end
+
+    it "handles multiple JSON mime types" do
+      json_mime_types = [
+        "application/json",
+        "application/x-json",
+        "text/json",
+        "text/x-json",
+        "+json"
+      ]
+      request = build_request
+      body = {result: "value"}.to_json
+
+      json_mime_types.each do |mime_type|
+        response = build_response(
+          status: 200,
+          content_type: mime_type,
+          body: body
+        )
+        serializer = Readme::Har::ResponseSerializer.new(
+          request,
+          response,
+          Filter.for
+        )
+        json = serializer.as_json
+
+        expect(json[:content][:text]).to eq body
+        expect(json[:content][:mimeType]).to eq mime_type
+      end
+    end
```

Comment:
This might be a rebase issue? I removed this in the previous commit as this behavior moved to another object

Reply:
Nice catch, I definitely added this back when I rebased.